polaris/hub/client.py

```diff
@@ -751,19 +752,17 @@ def _upload_benchmark(
         )
         return response
 
-    
+
     def upload_competition(
-        self, 
-        dataset: Dataset, 
```

Comment:
Removed the reference to the dataset here, we don't need it.

Reply:
Nice catch!